src/OrcasEngine/Resources/Constants.cs

```diff
@@ -87,10 +87,14 @@ internal static string AssemblyVersion
         {
             get
             {
-                Version version = new Version(ThisAssembly.AssemblyVersion);
+#if STANDALONEBUILD
```

Comment:
I copied what was in Build.Engine's [Resources/Constants.cs](https://github.com/Microsoft/msbuild/blob/master/src/XMakeBuildEngine/Resources/Constants.cs#L159). The error was in ThisAssembly. ThisAssembly is defined in the DevDiv build so this is wrong for our OSS build. I think GitVersioning also defines ThisAssembly but if that's turned off in netcore we don't want to use it anyway. I'm not positive this is right, but it is what we're doing in Microsoft.Build.dll.